requirements.txt

```diff
@@ -25,3 +23,13 @@ requests==2.28.1
 Shapely==1.8.5.post1
 SQLAlchemy~=1.4.0
 urllib3==1.26.13
+
+# Test requirements
+black~=22.12.0
+coverage~=7.0.1
+flake8~=6.0.0
+flake8-docstrings~=1.6.0
+flake8-pyproject~=1.2.2
+isort~=5.11.4
+mypy==0.991
+pytest==7.2.0
```

Comment:
py-test staat er hierboven nog een keer in.

Reply:
fixed 👍